NEWS

```diff
@@ -45,8 +51,8 @@ Any uppercase BUG_* names are modernish shell bug IDs.
 
 - A two decade old bug that caused 'whence -a' to base the path of
   tracked aliases on the user's current working directory has been
-  fixed. Now the real path to the tracked aliases is shown when '-a'
-  is passed to the whence command.
+  fixed. Now the real path to tracked aliases is shown when '-a' is
+  passed to the whence command.
```

Comment:
You might have noticed that, in `README.md`, I linked the `NEWS` file to GitHub's version of `git blame`, so that people can easily associate news items with commits revealing the full nitty gritty. So I would prefer to have no trivial edits of items that are good enough, as that would confuse this system. If there are serious errors in `NEWS`, they should be fixed in their own commits.

Reply:
That makes sense. The typo fix is no longer a part of the pull request.